src/ssl/cert_validate_message.cc

```diff
@@ -65,7 +65,8 @@ Ssl::CertValidationMsg::composeRequest(CertValidationRequest const &vcert)
         int i = 0;
         for (const Security::CertErrors *err = vcert.errors; err; err = err->next, ++i) {
             body +="\n";
-            body = body + param_error_name + xitoa(i) + "=" + GetErrorName(err->element.code) + "\n";
+            const char *errName = GetErrorName(err->element.code);
+            body = body + param_error_name + xitoa(i) + "=" + (errName ? errName : xitoa(err->element.code)) + "\n";
```

Comment:
If this code stays: We should not use two xitoa() calls in one statement even though it "works". That function uses static storage. Let's split into two statements.